PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,21 +1,34 @@
-High level description including:
- - What does this change enable
- - What bugs does this change fix
- - High-Level how things changed
+___
+**What is the review process and when will my changes land?**
 
-# Issues
+All PR's require 2 approvals using github's [pull request reviews](https://help.github.com/articles/about-pull-request-reviews/).
 
-[DCOS-<number>](https://dcosjira.atlassian.net/browse/DCOS-<number>)
-...
+Reviewers should be:
+* Developers who understand the code being modified.
+* Developers responsible for code that interacts with or depends on the code being modified.
 
-# Checklist
+It is best to proactively ask for 2 reviews by mentioning the candidate reviewers in the PR comments area. The responsibility is on the developer submitting the PR to follow-up with reviewers and make sure a PR is reviewed in a timely manner. Once a PR has **2 ship-it's**, **no red reviews**, and **all tests are green** it will be included in the next train. Trains usually depart the station within 1 business day of landing.
 
- - [ ] Included a test which will fail if code is reverted but test is not
- - [ ] Followed relevant code rules [Rules for Packages and Systemd](https://github.com/dcos/dcos/tree/master/docs)
+**PLEASE FILL IN THE TEMPLATE BELOW** / **DO NOT REMOVE ANY SECTIONS BELOW THIS LINE**
+___
+## High Level Description
 
-If this is a component/package update, include
+What features does this change enable? What bugs does this change fix? High-Level description of how things changed.
 
- - [ ] Change Log from last: <link>
- - [ ] Test Results: <link>
- - [ ] Code Coverage: <link>
+## Related Issues
 
+  - [DCOS-<number>](https://dcosjira.atlassian.net/browse/DCOS-<number>)
```

Comment:
I'd also love to see the JIRA title/summary on this line, so I don't have to hover to see what each one is.
```
[DCOS-<number>](https://dcosjira.atlassian.net/browse/DCOS-<number>) Foo the Bar so it stops Bazzing.
[DCOS-<number>](https://mesosphere.atlassian.net/browse/DCOS-<number) TRACKING: dcosjira-123 Foo the Bar
```